src/cgame/rocket/rocket_documents.cpp

```diff
@@ -168,3 +169,30 @@ void Rocket_DocumentAction( const char *name, const char *action )
 		}
 	}
 }
+
+static bool debugger_initialised = false;
+void CG_RmlUIDebuggerToogle()
+{
+	if (!debugger_initialised)
+	{
+		Rml::Debugger::Initialise(*cgs.mapname // CHECKME
```

Comment:
Also, once you pick  a context, are you locked into that context forever?

Reply:
That's the best I can do with current RmlUI, yes. More recent versions have introduced `Rml::Debugger::Shutdown();` which would allow switching

Reply:
Rml::Debugger::SetContext still exists and probably works.

Reply:
Looking deeper, it seems we always want to initialize the debugger into the menuContext and then allow switching between the two via `debuggerToggle menu` or `debuggerToggle hud`. Still unsure if debuggerToggle will work with the hudContext though.

Reply:
For now, let's just enable the debugger for the menuContext and leave the hudContext as a TODO. This should make the code easier for now.